pkg/kubectl/cmd/apiversions.go

```diff
@@ -51,19 +51,13 @@ func RunApiVersions(f *cmdutil.Factory, w io.Writer) error {
 		return err
 	}
 
-	apiVersions, err := client.ServerAPIVersions()
+	groupList, err := client.Discovery().ServerGroups()
```

Comment:
Good point. We need to fix the ServerGroups to not return error if the apiserver is 1.0.